docs/content/docs/user-docs/leader-election.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Leader election"
+description: "Configure leader election for ACK controllers"
+lead: "How to enable leader election for ACK controllers"
+draft: false
+menu:
+  docs:
+    parent: "getting-started"
+weight: 60
+toc: true
+---
+
+In a Kubernetes cluster, multiple instances of a controller maybe running
+simultaneously to manage resources and perform tasks. However, to avoid conflicts
+and ensure proper resource management, it's necessary to designate one instance
+as the leader, which takes responsibility for executing certain operations while
+the other instances remain passive. In the event that the leader instance fails,
+[leader election][leader-election] ensures the seamless transition of leadership
+to another healthy instance.
+
+In ACK land, by default, the feature of leader election is disabled. To utilize
```

Comment:
I'm not a fan of `In ACK land` lol. I'd just say `By default, the ACK Helm charts disable leader election`